zap/src/main/installer/resources/Messages.properties

```diff
@@ -9,14 +9,14 @@ zap.installer.cfu.screensubtitle = Configure how ${compiler:sys.fullName} should
 zap.installer.cfu.title = Check for Updates on startup
 zap.installer.cfu.releases.title = ZAP Releases
 zap.installer.cfu.releases.autodownload = Automatically download new ZAP releases
-zap.installer.cfu.addonupdates.title = Add-on updates
-zap.installer.cfu.addonupdates.check = Check for updates to the add-ons you have installed
-zap.installer.cfu.addonupdates.autoinstall = Automatically install updates to the add-ons you have installed
-zap.installer.cfu.addonupdates.autoinstallscanners = Automatically install updates to the scanner rules you have installed
+zap.installer.cfu.addonupdates.title = Add-on Updates
+zap.installer.cfu.addonupdates.check = Check for Updates to the Add-ons you have installed
+zap.installer.cfu.addonupdates.autoinstall = Automatically install Updates to the Add-ons you have installed
+zap.installer.cfu.addonupdates.autoinstallscanners = Automatically install Updates to the scanner rules you have installed
```

Comment:
Why these changes?

Reply:
I was not sure about the instructions and made changes randomly 😔 sorry for that. I was using intillij as IDE and had no idea how to check the gui for these changes.

Reply:
Well since they're installer strings you'd have to actually build the installer and check.

I plan to have a look but won't get to it until tomorrow night.

There's developer docs here that'd probably help you: https://www.zaproxy.org/docs/developer/